app/controllers/api/physical_storages_controller.rb

```diff
@@ -12,6 +12,14 @@ def refresh_resource(type, id, _data = nil)
       end
     end
 
+    def create_resource(_type, _id = nil, data = {})
+      ext_management_system = ExtManagementSystem.find(data['ems_id'])
+      task_id = PhysicalStorage.create_physical_storage_queue(session[:userid], ext_management_system, data)
+      action_result(true, 'Creating Physical Storage', :task_id => task_id)
```

Comment:
might be nice to return in the action result the target in addition to the  generic message, maybe:

```
Creating Physical Storage #{data["name"]} for Provider id: #{data["ems_id"]}
```